qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateDeploymentTest.java

```diff
@@ -157,4 +166,35 @@ public void shouldRejectDeployIfResourceIsTooLarge() {
         .isInstanceOf(ClientException.class)
         .hasMessageContaining("rejected with code 'EXCEEDED_BATCH_RECORD_SIZE'");
   }
+
+  @Test
+  public void shouldNotWriteResourcesInformationInRejectedRecords() {
+    // when
+    final var modelThatFitsJustWithinMaxMessageSize =
+        Bpmn.createExecutableProcess("PROCESS")
+            .startEvent()
+            .documentation("x".repeat((1046900)))
+            .done();
+    final var command =
```

Comment:
:wrench: @nicpuppa I guess we don't need the `final var command` in the test you added. It doesn't impact anything, but might be nice to remove it so the code-scan doesn't complain.